sensors/nxt_i2c_sensor_defs.c

```diff
@@ -262,6 +262,19 @@ static int ht_angle_scale(void *context,
 	return 0;
 }
 
+static int ms_pps58_nx_set_mode_post_cb(nxt_i2c_sensor_data *sensor, u8 mode)
+{
+	struct i2c_client *client = sensor->client;
+	int ref;
+
+	if(mode < 4 || 6 < mode)
+		return 0;
+
+	ref = i2c_smbus_read_word_data(client, 0x43);
+	if(ref < 0)
+		return ref;
+	return i2c_smbus_write_word_data(client, 0x47, ref);
```

Comment:
I'm not sure about this. This would change the behaviour of the ZERO command as well. The notion that I have about this is pretty much the same as how a mass scale is used - the scale is zeroed at first use to get a reference weight.

Either way it should be possible to set the reference to some value (either the current absolute pressure or some arbitrary value). I don't see that this is possible as it stands. What should I add to do that?